src/index.md

```diff
@@ -11,6 +11,9 @@ generics and error handling.
 > The latest version of the course can be found at
 > <https://google.github.io/comprehensive-rust/>. If you are reading somewhere
 > else, please check there for updates.
+>
+> The course is also available
+> [as a PDF](https://google.github.io/comprehensive-rust/comprehensive-rust.pdf).
```

Comment:
I think a relative link will work better: it should be correct for all languages out of the box:

```suggestion
> [as a PDF](comprehensive-rust.pdf).
```

Reply:
I wasn't sure if that'd be an issue when the PDF isn't available (e.g., building the book locally with `mdbook build`). Is it fine if the link is broken in such cases?

Reply:
Good question... I would say yes, building things locally is something people can (and should) do while developing the material, but the publishing workflow is the one that actually counts.

The `build.sh` script is a little step towards formalizing this. @sakex has been working on #1236, which will let `mdbook` do all the work for all the translations.